views/result_selection.py

```diff
@@ -194,21 +197,34 @@ def select_ts_datasource(self):
             # Little test for checking if there is an id mapping file available
             # If not we check if an .h5 file is available
             # If not we're not going to proceed
-            try:
-                find_id_mapping_file(filename)
-                ds_type = 'netcdf'
-            except IndexError:
+
+            if detect_netcdf_version(filename) == 'netcdf-groundwater':
                 try:
                     find_h5_file(filename)
                     ds_type = 'netcdf-groundwater'
                 except IndexError:
-                    pop_up_info("No id mapping or .h5 file found, we tried the following "
-                                "locations: id_mapping in [., ../input_generated] and"
-                                "h5_file in [., ../preprocessed]. Please add "
-                                "this file to the correct location and try again.",
+                    pop_up_info("You selected a netcdf that was created "
```

Comment:
Nice. But I think it would be even better if this message contains more information. Can you include something about groundwater support?